lib/core/src/animation/animator/fixed_step.rs

```diff
@@ -15,11 +15,14 @@ use crate::system::web::animation_frame_loop::AnimationFrameLoop;
 /// This struct counts the intervals in a time period.
 #[derive(Debug)]
 pub struct IntervalCounter {
+    /// Interval duration.
     pub interval_duration : f32,
+    /// The amount of time accumulated by `IntervalCounter`'s `add_time`.
     pub accumulated_time  : f32
 }
 
 impl IntervalCounter {
+    /// Creates `IntervalCounter` which counts how many intervals with an `interval_duration`.
```

Comment:
I'm ok with it. Just FYI - the function `new` I always just document as `/// Constructor.` because it is abvious what it does. However, I put bigger emphasis on documenting the structure itself. The information about counting intervals etc - describes what the structure does and how it operates, it does not strictly documents the `new` function itself.

Reply:
@wdanilo I think it's interesting to document `new` as it takes some parameters which only makes sense during its construction.